src/simsopt/field/biotsavart.py

```diff
@@ -2,6 +2,53 @@
 import simsoptpp as sopp
 from simsopt.field.magneticfield import MagneticField
 from simsopt.geo.curve import Curve
+from simsopt._core.graph_optimizable import Optimizable
+from simsopt._core.derivative import Derivative
+
+
+class Coil(sopp.Coil, Optimizable):
+
+    def __init__(self, curve, current):
+        self.__curve = curve
+        self.__current = current
```

Comment:
Any reason for double __?

Reply:
I'm using this to indicate that they are private variables that shouldn't be touched by the user. I think double underscore is the pythonic way to do that.